frontend/src/component/feature/FeatureForm/FeatureForm.tsx

```diff
@@ -197,6 +197,7 @@ const FeatureForm: React.FC<IFeatureToggleForm> = ({
                     </StyledInputDescription>
                 }
             />
+            // TODO: this can be removed after new create flag flow goes live
```

Comment:
Same thing; add the flag name:
```suggestion
            // TODO: this can be removed after new create flag flow goes live. Flag: improveCreateFlagFlow
```